package.json

```diff
@@ -78,7 +78,7 @@
     "pre_test": "npm run lint && npm run docker_test",
     "tape": "node test/tape/index.js | tap-spec",
     "debug_tape": "node-debug test/tape/index.js",
-    "test": "npm run pre_test && istanbul --config=test/.istanbul.yml cover node_modules/mocha/bin/_mocha -- --check-leaks -t 10000 -b -R spec test/index.js && npm run tape"
+    "test": "node_modules/mocha/bin/mocha --inspect --check-leaks -t 10000 -b -R spec test/index.js"
```

Comment:
Why you have removed coverage and tape tests?

Reply:
This is just a progress commit. I'll ping you when it's ready.